data-model/fdo-types/0.3.0/digital-specimens/schema/occurrences.json

```diff
@@ -0,0 +1,256 @@
+      "examples": [
+        "present"
+      ],
+      "enum": [
+        "present",
+        "absent"
+      ]
+    },
+    "dwc:pathway": {
+      "type": "string",
+      "description": "https://rs.tdwg.org/dwc/terms/pathway",
+      "examples": [
+        "releasedForUse"
+      ],
+      "$comment": "Could be an enumeration based on https://rs.tdwg.org/dwc/doc/pw/"
+    },
+    "dwc:degreeOfEstablishment": {
+      "type": "string",
+      "description": "https://rs.tdwg.org/dwc/terms/degreeOfEstablishment",
+      "examples": [
+        "captive"
+      ],
+      "$comment": "Could be an enumeration based on https://rs.tdwg.org/dwc/doc/doe/"
+    },
+    "dwc:georeferenceVerificationStatus": {
+      "type": "string",
+      "description": "https://rs.tdwg.org/dwc/terms/georeferenceVerificationStatus",
+      "examples": [
+        "verified by data custodian"
+      ],
+      "$comment": "Could be an enumeration based on https://rs.tdwg.org/dwc/doc/em/"
+    },
+    "dwc:occurrenceRemarks": {
+      "type": "string",
+      "description": "https://rs.tdwg.org/dwc/terms/occurrenceRemarks",
+      "examples": [
+        "found dead on road"
+      ]
+    },
+    "ods:eventName": {
+      "type": "string",
+      "description": "The name of the event"
+    },
+    "dwc:fieldNumber": {
+      "type": "string",
+      "description": "https://rs.tdwg.org/dwc/terms/fieldNumber",
+      "examples": [
+        "RV Sol 87-03-08"
+      ]
+    },
+    "dwc:recordNumber": {
+      "type": "string",
+      "description": "https://rs.tdwg.org/dwc/terms/recordNumber",
+      "examples": [
+        "OPP 7101"
+      ]
+    },
+    "dwc:eventDate": {
+      "type": "string",
+      "description": "https://rs.tdwg.org/dwc/terms/eventDate",
+      "examples": [
+        "1963-03-08T14:07-0600"
+      ]
+    },
+    "dwc:verbatimEventDate": {
+      "type": "string",
+      "description": "https://rs.tdwg.org/dwc/terms/verbatimEventDate",
+      "examples": [
+        "Marzo 2002"
+      ]
+    },
+    "dwc:year": {
+      "type": "integer",
+      "description": "https://rs.tdwg.org/dwc/terms/year",
+      "maximum": 9999,
+      "examples": [
+        2008
+      ]
+    },
+    "dwc:month": {
+      "type": "integer",
+      "description": "https://rs.tdwg.org/dwc/terms/month",
+      "maximum": 12,
+      "minimum": 1,
+      "examples": [
+        12
+      ]
+    },
+    "dwc:day": {
+      "type": "integer",
+      "description": "https://rs.tdwg.org/dwc/terms/day",
+      "maximum": 31,
+      "minimum": 1,
+      "examples": [
+        29
+      ]
+    },
+    "dwc:habitat": {
+      "type": "string",
+      "description": "https://rs.tdwg.org/dwc/terms/habitat",
+      "examples": [
+        "savanna"
+      ],
+      "$comment": "Is this necessary for specimens"
+    },
+    "eco:protocolDescriptions": {
+      "type": "string",
+      "description": "https://rs.tdwg.org/eco/terms/protocolDescriptions",
```

Comment:
this link doesn't resolve. what's this namespace?

Reply:
are you sure? It works for me if I control-click it from InteliJ

Reply:
nvm must have copied the link wrong!